snake.py

```diff
@@ -0,0 +1,61 @@
+from turtle import Turtle
+STARTING_POSITIONS = [(0, 0), (-20, 0), (-40, 0)]
```

Comment:
When you stash at the top, you can use a mapping and then all you have to update is that.  That also allows you to stash in a JSON (instead of `.py` code), which means that you won't have to run a deploy if you want to make changes.  That's how you can make the code super flexible.

Reply:
imported json file for constants